### PR TITLE
Detailed rework of the first thousand lines of dialogue

### PR DESCRIPTION
**Pull request by A3D-Anarial**
Also contains some low-risk find-and-replace operations to remove leading spaces before exclamation marks and question marks.

### CODE REVIEW

File: MWStoryGitFile.txt [hunk 1]
```diff
@@ -27308,7 +27308,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Sep 5 14:35:08 GMT+0200 2021_16770"MainText""
 "Sun Sep 5 14:35:08 GMT+0200 2021_16774"MainText"A device with much happy fluid inside for vaginal cleaning use."
 "Sun Sep 5 14:35:08 GMT+0200 2021_16778"MainText"This... ahm..."
-"Sun Sep 5 14:35:08 GMT+0200 2021_16778"If_story_text"You want to fill me up?"
+"Sun Sep 5 14:35:08 GMT+0200 2021_16778"If_story_text"You want to fill meup?"
```
Comment (KleinKerni) [inline MWStoryGitFile.txt]: You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

File: MWStoryGitFile.txt [hunk 2]
```diff
@@ -24926,7 +24926,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Jul 11 15:57:18 GMT+0200 2021_15007"MainText"First I would like to see it!"
 "Sun Jul 11 15:57:18 GMT+0200 2021_15008"MainText"Alright, we better build up this device in the bathroom."
 "Sun Jul 11 15:57:18 GMT+0200 2021_15009"MainText"This... I think is not a good idea. I barely have any space left in there."
-"Sun Jul 11 15:57:18 GMT+0200 2021_15010"MainText"Okay... then... ahm... do you have something to put underneath it? Something water resistant?"
+"Sun Jul 11 15:57:18 GMT+0200 2021_15010"MainText"Okay... then... ahm... do you have something to put underneath it? Something waterresistant?"
```
Comment (KleinKerni) [inline MWStoryGitFile.txt]: You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

File: MWStoryGitFile.txt [hunk 3]
```diff
@@ -23199,7 +23199,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sat May 8 15:54:24 GMT+0200 2021_14455"MainText"She is on the medical bed and moans all the time, like... you know..."
 "Sat May 8 15:54:24 GMT+0200 2021_14456"MainText"Hehe... having fun with a toy I assume?"
 "Sat May 8 15:54:24 GMT+0200 2021_14457"MainText"I do not know... no she did not played just moaned."
-"Sat May 8 15:54:24 GMT+0200 2021_14458"MainText"Sounds really weird actually. Shall we go and have a look?"
+"Sat May 8 15:54:24 GMT+0200 2021_14458"MainText"Sounds really weird actually. Shall we go and have alook?"
```
Comment (KleinKerni) [inline MWStoryGitFile.txt]: You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

File: MWStoryGitFile.txt [hunk 4]
```diff
@@ -20444,7 +20444,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Nov 29 14:17:28 GMT+0100 2020_12864"MainText""
 "Sun Nov 29 14:17:28 GMT+0100 2020_12867"MainText"You want this, right?"
 "Sun Nov 29 14:17:28 GMT+0100 2020_12870"MainText"You ... the... "
-"Sun Nov 29 14:17:28 GMT+0100 2020_12870"If_story_text"You want to fill me up?"
+"Sun Nov 29 14:17:28 GMT+0100 2020_12870"If_story_text"You want to fill meup?"
```
Comment (KleinKerni) [inline MWStoryGitFile.txt]: You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

File: MWStoryGitFile.txt [hunk 5]
```diff
@@ -19880,7 +19880,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Nov 8 16:33:20 GMT+0100 2020_12231"MainText""
 "Sun Nov 8 17:05:01 GMT+0100 2020_12009"MainText"And now for a very special feature, something that Malia loves."
 "Sun Nov 8 17:05:01 GMT+0100 2020_12010"MainText"I love it too!"
-"Sun Nov 8 17:05:01 GMT+0100 2020_12011"MainText"You know what I am talking about?"
+"Sun Nov 8 17:05:01 GMT+0100 2020_12011"MainText"You know what I am talkingabout?"
```
Comment (KleinKerni) [inline MWStoryGitFile.txt]: You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<

**Review by KleinKerni — CHANGES_REQUESTED** [review]
Please have a look underneath your main changes , it seems that you re-introduce some spelling errors which have already been resolved beforehand.

File: MWStoryGitFile.txt (hunk 1 above)
Comment (A3D-Anarial) [reply MWStoryGitFile.txt]: I suspected that might happen, yes, unfortunately their changes conflicted with mine to a fairly significant extent, and I had roughly 2750 individual conflicts to resolve. I used my IDE's "resolve simple conflicts" function to process the bulk of them and it seems as though it didn't get it right every time. I'll go through and fix up the ones that you've spotted so far, there may be more to find.

Comment (A3D-Anarial) [reply MWStoryGitFile.txt]: Should all be resolved now

**Review by KleinKerni — APPROVED** [review]

**Pull request merged**